end-to-end_sales-forecasting/sales_forecasting-end-to-end.ipynb

```diff
@@ -0,0 +1,2701 @@
+{
```

Comment:
<p>Can we download from pypi rather than whl on downloads page?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/truera/truera-examples/pull/21/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Yes, will update this. </p>